### PR TITLE
ELEMENTS-1577: meaningful labels associated with item name for favori…

### PR DESCRIPTION
**Pull request by swarnadipa-dev**
…tes and download button

### CODE REVIEW

**Review by rakeshkumar1019 — DISMISSED** [review]

**Review by satishyadav-dev — DISMISSED** [review]

**Review by Nishant0928 — DISMISSED** [review]

File: ui/actions/nuxeo-favorites-toggle-button.js [hunk 1]
```diff
@@ -110,7 +114,7 @@ import '../nuxeo-button-styles.js';
 
         _label: {
           type: String,
-          computed: '_computeLabel(favorite, i18n)',
+          computed: '_computeLabel(favorite, document, i18n)',
```
Comment (rajatgoel-gl) [inline ui/actions/nuxeo-favorites-toggle-button.js]: This should be _computeHoverLabel

File: ui/actions/nuxeo-download-button.js [hunk 2]
```diff
@@ -97,7 +101,7 @@ import '../nuxeo-button-styles.js';
 
         _label: {
           type: String,
-          computed: '_computeLabel(i18n)',
+          computed: '_computeLabel(document, i18n)',
```
Comment (rajatgoel-gl) [inline ui/actions/nuxeo-download-button.js]: This should be _computeHoverLabel

**Review by rajatgoel-gl — CHANGES_REQUESTED** [review]

File: ui/actions/nuxeo-favorites-toggle-button.js (hunk 1 above)
Comment (swarnadipa-dev) [reply ui/actions/nuxeo-favorites-toggle-button.js]: removed the extra param. function name is correct.

File: ui/actions/nuxeo-download-button.js (hunk 2 above)
Comment (swarnadipa-dev) [reply ui/actions/nuxeo-download-button.js]: removed the extra param. function name is correct.

**Review by rakeshkumar1019 — APPROVED** [review]

**Review by Nishant0928 — APPROVED** [review]

**Review by rajatgoel-gl — APPROVED** [review]

**Pull request merged**